src/components/content/order/serviceActions/CurrentServiceActions.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { Alert, Card, Empty, Tabs } from 'antd';
+import React, { useState } from 'react';
+import '../../../../styles/app.module.css';
+import serviceActionStyles from '../../../../styles/service-action.module.css';
+import {
+    DeployedServiceDetails,
+    UserOrderableServiceVo,
+    VendorHostedDeployedServiceDetails,
+} from '../../../../xpanse-api/generated';
+import { isHandleKnownErrorResponse } from '../../common/error/isHandleKnownErrorResponse';
+import { ServiceActionParameter } from './ServiceActionParameter';
+import { useCreateServiceActionRequest } from './useCreateServiceActionRequest';
+
+export const CurrentServiceActions = ({
+    deployedService,
+    userOrderableServiceVo,
+}: {
+    deployedService: DeployedServiceDetails | VendorHostedDeployedServiceDetails;
+    userOrderableServiceVo: UserOrderableServiceVo | undefined;
+}): React.JSX.Element => {
+    const createServiceActionRequest = useCreateServiceActionRequest();
+    const [actionName, setActionName] = useState<string | undefined>(userOrderableServiceVo?.serviceActions?.[0]?.name);
+    const onChange = (key: string) => {
+        setActionName(key);
+        createServiceActionRequest.reset();
+    };
+    const items =
+        userOrderableServiceVo?.serviceActions?.map((serviceAction) => {
+            return {
+                key: serviceAction.name,
+                label: serviceAction.name,
+                children: (
+                    <ServiceActionParameter
+                        serviceId={deployedService.serviceId}
+                        actionName={serviceAction.name}
+                        actionParameters={serviceAction.actionParameters ?? []}
+                        createServiceActionRequest={createServiceActionRequest}
+                    />
+                ),
+            };
+        }) ?? [];
+
+    return (
+        <>
+            {createServiceActionRequest.isSuccess ? (
+                <Alert
+                    className={serviceActionStyles.serviceActionAlert}
+                    message={'Create Service Action Successfully'}
+                    description={
+                        <>
+                            Create Service Action with actionName <strong>{actionName}</strong> Successfully.
```

Comment:
description can be - use the orders page in MyServices to see the status of the order. 
Also show the order id. 

service configuration must also show the order Id it's not done already.